ipaserver/install/server/replicainstall.py

```diff
@@ -738,6 +775,7 @@ def install_check(installer):
                          ldap_uri=ldapuri)
     remote_api.finalize()
     installer._remote_api = remote_api
+
```

Comment:
It is still there: https://github.com/freeipa/freeipa/pull/437/files#diff-4fc382c22d987062a45dddeaac8bb461L648